project_buttom_kanban_open_form_view/README.rst

```diff
@@ -0,0 +1,60 @@
+====================================
+Proyect buttom kanban open form view
```

Comment:
I dont like this name.. BUT it can be... project_direct_access_form_in_kanban_view but dont like too.. @dalonsod what do you propose?

Reply:
Well, try this: `project_kanban_form_direct_access`.